pydra/engine/core.py

```diff
@@ -539,10 +538,11 @@ def __getattr__(self, name):
         return self.__getattribute__(name)
 
     @property
-    def done(self):
-        if super(Workflow, self).done:
-            return True
-        # TODO: not sure why this is needed
+    def done_all_tasks(self):
```

Comment:
i would rename this to `done`

Reply:
but this is the problem it's different than `done`. We still have `done` property in the base class

Reply:
ok